alfred-api-docker/234-elasticsearch/docker-compose.yml

```diff
@@ -0,0 +1,166 @@
+services:
+  alfresco-core:
+    image: ${DOCKER_IMAGE:-image_not_set}
+    ports:
+      - "8080:8080"
+      - "4578:4578" # coderunner-server
+    volumes:
+      - alfresco:/opt/alfresco/alf_data
+    restart: unless-stopped
+    environment:
+      - TERM=xterm
+      - TOMCAT_ALLOW_CASUAL_MULTIPART_PARSING=false
```

Comment:
- TERM=xterm
      - TOMCAT_ALLOW_CASUAL_MULTIPART_PARSING=false

Can you explain these environment settings?
Are these best practice?
Should we be applying them elsewhere?